src/main/java/de/tum/in/www1/artemis/web/rest/ResultResource.java

```diff
@@ -51,6 +51,7 @@
     private final FeedbackService feedbackService;
     private final UserService userService;
     private final ContinuousIntegrationService continuousIntegrationService;
+    private final CourseService courseService;
```

Comment:
I think we can revert changes in this file?

Reply:
Done